src/node_worker.cc

```diff
@@ -46,6 +47,8 @@ namespace node {
 namespace worker {
 
 constexpr double kMB = 1024 * 1024;
+std::unordered_map<uint64_t, Worker*> workers;
+MessagePort* main_port;
```

Comment:
Is there a reason for keeping `main_port` outside of the map? I would assume the main thread doesn't really differ from workers and `thread_id: 0` would fit well into the `workers` map.

Reply:
I don't think there is a `Worker` for the main thread. @joyeecheung Am I right?

Reply:
You could use `std::unordered_map<uint64_t, MessagePort*>` as you don't need the full `Worker *`.

Reply:
Thanks, it was so obvious! :) Changed locally!